### PR TITLE
Optimize data retention policies

### PR DESCRIPTION
**Pull request by wenchajun**
fix https://github.com/kubesphere/kubesphere/issues/4604
Signed-off-by: chengdehao <wenchajun@gmail.com>

### CODE REVIEW

**Comment by wanjunlei** [comment]
If `common.logging.logMaxAge` changed, will the curator config file change as well?

**Comment by wenchajun** [comment]
> If `common.logging.logMaxAge` changed, will the curator config file change as well?

In my tests it changes. It may be that under the `common` field, each modification will upgrade once. Please refer to: https://github.com/kubesphere/ks-installer/blob/master/roles/common/tasks/es-install.yaml#L135
I don't know if this is the correct explanation. @pixiake Could you help to see if this is the right thing to do?

File: roles/common/templates/custom-values-elasticsearch-curator.yaml.j2 [hunk 1]
```diff
@@ -35,7 +35,7 @@ configMaps:
     actions:
       1:
         action: delete_indices
-        description: "Clean up ES by deleting old indices"
+        description: "Clean up ES by deleting all old indices"
```
Comment (wanjunlei) [inline roles/common/templates/custom-values-elasticsearch-curator.yaml.j2:38]: Why need to keep the task that deleting all old indices?

**Comment by wanjunlei** [comment]
The log, event, and audit retention policies will be exposed through the UI. The current method requires re-running the installer every time it is modified, which is unreasonable. We need to extract these parameters and save them separately to achieve fast update operations.

**Review by wanjunlei — APPROVED** [review]

**Review by pixiake — COMMENTED** [review]
/lgtm

**Pull request merged**